src/core/tests/functional/test_config.py

```diff
@@ -272,3 +272,144 @@ def test_delete_report_item_type(self, client, auth_header, cleanup_report_item_
+    base_uri = "/api/config"
+
+    def test_create_publisher_preset(self, client, auth_header, cleanup_publisher_preset):
+        response = self.assert_post_ok(client, uri="publishers-presets", json_data=cleanup_publisher_preset, auth_header=auth_header)
+        assert response.json["message"] == "Publisher preset created successfully"
+        assert response.json["id"] == "44"
+
+    def test_modify_publisher_preset(self, client, auth_header, cleanup_publisher_preset):
+        publisher_data = {"description": "new description"}
+        publisher_preset_id = cleanup_publisher_preset["id"]
+        response = self.assert_put_ok(
+            client, uri=f"publishers-presets/{publisher_preset_id}", json_data=publisher_data, auth_header=auth_header
+        )
+        # TODO: again - successful modification returns None (null)
+        assert response.json is None
+
+    def test_get_publisher_preset(self, client, auth_header, cleanup_publisher_preset):
+        response = self.assert_get_ok(client, uri=f"publishers-presets?search={cleanup_publisher_preset['name']}", auth_header=auth_header)
+        assert response.json["items"][0]["name"] == cleanup_publisher_preset["name"]
+        assert response.json["items"][0]["id"] == cleanup_publisher_preset["id"]
+        assert response.json["items"][0]["description"] == "new description"
+        assert response.json["items"][0]["type"] == cleanup_publisher_preset["type"]
+        assert response.json["items"][0]["parameters"]["FTP_URL"] == cleanup_publisher_preset["parameters"]["FTP_URL"]
+
+    def test_delete_publisher_preset(self, client, auth_header, cleanup_publisher_preset):
+        publisher_preset_id = cleanup_publisher_preset["id"]
+        response = self.assert_delete_ok(client, uri=f"publishers-presets/{publisher_preset_id}", auth_header=auth_header)
+        assert response.json["message"] == "PublisherPreset 44 deleted"
+
+
+class TestAttributes(BaseTest):
+    base_uri = "/api/config"
+
+    def test_create_attribute(self, client, auth_header, cleanup_attribute):
+        response = self.assert_post_ok(client, uri="attributes", json_data=cleanup_attribute, auth_header=auth_header)
+        assert response.json["id"] == 42
+        assert response.json["message"] == "Attribute added"
+
+    def test_modify_attribute(self, client, auth_header, cleanup_attribute):
+        attribute_data = {"name": "Attributify McAttributeFace"}
+        attribute_id = cleanup_attribute["id"]
+        response = self.assert_put_ok(client, uri=f"attributes/{attribute_id}", json_data=attribute_data, auth_header=auth_header)
+        # TODO really weird to return None (null)
+        assert response.json is None
+
+    def test_get_attribute(self, client, auth_header, cleanup_attribute):
+        attribute_id = cleanup_attribute["id"]
+        response = self.assert_get_ok(client, uri=f"attributes/{attribute_id}", auth_header=auth_header)
+        print(cleanup_attribute["type"])
```

Comment:
are prints needed?

Reply:
Unfortunately, logging isn't correctly configured yet, therefore Debug messages are printed on multiple occasions. 

Maybe we should create an issue to configure logging: https://docs.pytest.org/en/latest/how-to/logging.html#live-logs